internal/route/routes.go

```diff
@@ -51,19 +54,13 @@ func HandleIndex(w http.ResponseWriter, r *http.Request) {
 			panic(err)
 		}
 
-		u := fmt.Sprintf("/todo/%s", insertedTodos[0].ListID)
+		u := "/todo/" + insertedTodos[0].ListID
```

Comment:
Just a reminder that you don't have to use Sprintf. If it's a complex string or you're taking advantage of the formatting that it gives it's nice.

It can be faster to use concatenation actually. Also since it's a typed language it's less error-prone than you'd think to just `+` them together.